src/openvic-simulation/types/OrderedContainers.hpp

```diff
@@ -98,4 +99,67 @@ namespace OpenVic {
 		IsDequeOrderedMap<T> && std::same_as<Key, typename T::key_type> && std::same_as<Value, typename T::mapped_type>;
 	template<typename T, typename Key>
 	concept IsDequeOrderedSetOf = IsDequeOrderedSet<T> && std::same_as<Key, typename T::key_type>;
+
+	/* Case-Insensitive Containers */
+	struct case_insensitive_string_hash {
+		using is_transparent = void;
+
+	private:
+		/* Based on the byte array hashing functions in MSVC's <type_traits>. */
+		[[nodiscard]] static constexpr size_t _hash_bytes_case_insensitive(char const* first, size_t count) {
+			constexpr size_t _offset_basis = 14695981039346656037ULL;
+			constexpr size_t _prime = 1099511628211ULL;
```

Comment:
Probably should be:
```suggestion
			static constexpr size_t _offset_basis = 14695981039346656037ULL;
			static constexpr size_t _prime = 1099511628211ULL;
```

Reply:
Clangd says: `Definition of a static variable in a constexpr function is a C++23 extension`
I initially had them as `static constexpr` member variables of `case_insensitive_string_hash`, but seeing as they're only used in this function and only once each, I thought it simpler to just have them be local, I'd expect them to be subsituted in to the instructions that use them anyway (assign a `size_t` variable to `_offset_basis` and `*=` that variable by `_prime`).